### PR TITLE
added citation for nmf_itc paper

### PR DESCRIPTION
**Pull request by zthatch**

### CODE REVIEW

File: db/citations.yml [hunk 1]
```diff
@@ -10135,6 +10135,37 @@ thamp;prb13:
...
+    - Allison Wustrow
+    - Martin A. Karlsen
+    - James R. Neilson
+    - Dorthe B. Ravnsbæk
+    - Simon J. L. Billinge
+  doi: ''
+  entrytype: article
+  facility: nslsii, xpd, desy, P02.1
+  grant: efrc18
+  journal: arXiv:2112.13300 [cond-mat]
```
Comment (sbillinge) [inline db/citations.yml:10154]: this should be in `note`.  for the journal just put `arxiv`

File: db/citations.yml [hunk 2]
```diff
@@ -10135,6 +10135,37 @@ thamp;prb13:
...
+  month: dec
+  nb: ''
+  note: ''
+  number: ''
+  optannote: ''
+  optwwwlanl: ''
+  pages: '2112.13300'
+  title: 'nmfMapping: a cloud-based web application for non-negative matrix factorization
```
Comment (sbillinge) [inline db/citations.yml:10162]: this will have to be `{nmfMapping}` to compile correctly in latex

File: db/citations.yml [hunk 3]
```diff
@@ -10135,6 +10135,37 @@ thamp;prb13:
   volume: '88'
   wwwemail: ''
   year: '2013'
+thatc;arxiv21:
+  author:
```
Comment (sbillinge) [inline db/citations.yml:10139]: need an `ackno` (check the key) section with the acknowledgement copy pasted there (without rogue utf8's if possible...better to copy from the source....)

File: db/citations.yml [hunk 4]
```diff
@@ -10135,6 +10135,37 @@ thamp;prb13:
...
+  facility: nslsii, xpd, desy, P02.1
+  grant: efrc18
+  journal: arXiv:2112.13300 [cond-mat]
+  month: dec
+  nb: ''
+  note: ''
+  number: ''
+  optannote: ''
+  optwwwlanl: ''
+  pages: '2112.13300'
```
Comment (sbillinge) [inline db/citations.yml:10161]: need a `tags` item for building reading lists.   You can come up with anything, but ook at Tim's nmf paper and the PDFitc paper for examples.

Also, we need a `synopsis` item and a `professional_summary` and `public_summary` (new features but you should be able to find a couple of examples to go from)

**Review by sbillinge — COMMENTED** [review]
thanks!  Please see comments

File: db/citations.yml (hunk 4 above)
Comment (zthatch) [reply db/citations.yml:10173]: Let me know what you think of my `synopsis`, `professional_summary` and `public_summary`
  
I wasn't able to find an example of the public summary, but did my best to create what I would describe as a public summary.

File: db/citations.yml (hunk 3 above)
Comment (zthatch) [reply db/citations.yml:10151]: resolved

File: db/citations.yml (hunk 2 above)
Comment (zthatch) [reply db/citations.yml]: resolved

File: db/citations.yml (hunk 1 above)
Comment (zthatch) [reply db/citations.yml]: resolved

File: db/citations.yml [hunk 5]
```diff
@@ -10151,15 +10163,37 @@ thatc;arxiv21:
...
   month: dec
   nb: ''
-  note: ''
+  note: arXiv:2112.13300 [cond-mat]
   number: ''
   optannote: ''
   optwwwlanl: ''
   pages: '2112.13300'
-  title: 'nmfMapping: a cloud-based web application for non-negative matrix factorization
+  professional_summary: Non negative matrix factorization (NMF) has been shown previously
```
Comment (sbillinge) [inline db/citations.yml:10174]: perfect

File: db/citations.yml [hunk 6]
```diff
@@ -10151,15 +10163,37 @@ thatc;arxiv21:
...
+    to rapidly decompose large powder diffraction and atomic pair distribution function
+    (PDF) datasets into their constituent structural signals. Despite the utility
+    of NMF for quickly providing insight on large datasets, it is yet to gain widespread
+    adoption in the diffraction community. This is arguably due to a lack of a graphical
+    user interface or well written instructions to perform the analysis on diffraction
+    data. nmfMapping is presented in this paper as a web application that provides
+    an intuitive interface for exactly this purpose. This paper describes the interface
+    in detail, and provides two real examples of how the application has been used
+    to provide rapid insight into large in situ and operando datasets.
+  public_summary: Non negative matrix factorization can accelerate analysis of large
```
Comment (sbillinge) [inline db/citations.yml:10184]: for the public make it relevant to people's lives.    So something like "there is a need to find new materials for advanced applications such as batteries for electric vehicles.  Such materials are complex....." 

No jargon (in situ, operando, combinatorial diffraction...does a non-technical member of your family know or care what those are?)

File: db/citations.yml [hunk 7]
```diff
@@ -10151,15 +10163,37 @@ thatc;arxiv21:
...
+    datasets of in situ, operando, and combinatorial diffraction experiments by using
+    a machine learning algorithm to rapidly find and track varying structural signals.
+    The nmfMapping application has been launched on pdfitc.org in order to make this
+    technique more widely accessible to experimentalists who are analyzing PXRD or
+    PDF data.
+  synopsis: A web application called nmfMapping was created and launched on the pdfitc.org
+    website. This application provides a graphical interface for performing NMF decomposition
+    of powder x-ray diffraction (PXRD) and atomic pair distribtion function (PDF)
+    data. Examples of how the app has been used on real in situ and operando data
```
Comment (sbillinge) [inline db/citations.yml:10193]: can you make this last sentence less convoluted?   Decide what point you want to make then make the simplest sentence that makes that point.

File: db/citations.yml [hunk 8]
```diff
@@ -10151,15 +10163,37 @@ thatc;arxiv21:
...
+    are provided, along with a descriptions of the interface and the utility of NMF.
+  tags: in_situ, nmf
```
Comment (sbillinge) [inline db/citations.yml:10195]: I think some `pdf_method` type tags would be good too if you can find them already used.

**Review by sbillinge — COMMENTED** [review]
next round....

File: db/citations.yml (hunk 8 above)
Comment (zthatch) [reply db/citations.yml]: resolved

File: db/citations.yml (hunk 7 above)
Comment (zthatch) [reply db/citations.yml]: for review

File: db/citations.yml (hunk 6 above)
Comment (zthatch) [reply db/citations.yml]: updated for review

**Comment by sbillinge** [comment]
perfect @zthatch thanks!

**Pull request merged**